tests/util.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import os
+import shutil
+from textwrap import dedent
+
+from mock import patch
+
+from docutils.io import StringOutput
+
+from sphinx.builders.text import TextBuilder
+from sphinx.application import Sphinx
+from sphinx.environment import SphinxFileInput
+
+
+class MockSphinx(Sphinx):
```

Comment:
+1, I was thinking the same thing but didn't want to hinge anything on that just yet. Makes a ton of sense though.